dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemEmailNotifier.java

```diff
@@ -147,12 +151,38 @@ static public void sendRequest(Context context, RequestItem ri, String responseL
      * @param message email body (may be empty).
      * @throws IOException if sending failed.
      */
-    static public void sendResponse(Context context, RequestItem ri, String subject,
+    public void sendResponse(Context context, RequestItem ri, String subject,
             String message)
             throws IOException {
+        // Who granted this request?
+        List<RequestItemAuthor> grantors;
+        try {
+            grantors = requestItemAuthorExtractor.getRequestItemAuthor(context, ri.getItem());
+        } catch (SQLException e) {
+            LOG.warn("Failed to get grantor's name and address:  {}", e.getMessage());
+            grantors = List.of();
+        }
+
+        String grantorName;
+        String grantorAddress;
+        if (grantors.isEmpty()) {
+            grantorName = configurationService.getProperty("mail.admin.name");
+            grantorAddress = configurationService.getProperty("mail.admin");
+        } else {
+            RequestItemAuthor grantor = grantors.get(0); // XXX Cannot know which one
+            grantorName = grantor.getFullName();
+            grantorAddress = grantor.getEmail();
+        }
+
         // Build an email back to the requester.
-        Email email = new Email();
-        email.setContent("body", message);
+        Email email = Email.getEmail(I18nUtil.getEmailFilename(context.getCurrentLocale(),
+                "request_item.granted"));
+        email.addArgument(ri.getReqName()); // {0} requestor's name
+        email.addArgument(ri.getItem().getID().toString()); // {1} URL of the requested Item
+        email.addArgument(ri.getItem().getName()); // {2} title of the requested Item
+        email.addArgument(grantorName);     // {3} name of the grantor
+        email.addArgument(grantorAddress);  // {4} email of the grantor
```

Comment:
Remove this as noted.

Reply:
Again, why not make it available for those who want it, since the logic is worked out, but require a conscious effort to expose it in the email.